phpunit.xml.dist

```diff
@@ -24,6 +24,6 @@
         <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener"/>
     </listeners>
     <php>
-        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[total]=0"/>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak"/>
```

Comment:
@lstrojny I've seen you introduced this in https://github.com/lstrojny/phpunit-dom-assertions/commit/24a7db5b41d7c3deccd1a60c04f9cda1b8eaa487 but now only with `weak` I'm able to pass the tests in 8.0 with Symfony 4.

What should we do here?

Reply:
What is the error you are getting without weak?

Reply:
You can read it in the failing build of the previous commit: https://github.com/lstrojny/phpunit-dom-assertions/runs/1708224902

Reply:
A fix has landed in Symfony: https://github.com/symfony/symfony/commit/1f19da393645420ff53307da8d5d09c945e70f2d

Can you try setting the value to `max[direct]=0`?

Reply:
That seems to work. Now it would be elegant if we would set that only for the combination of PHP 8.0 and lowest dependencies.

Reply:
From https://symfony.com/doc/current/components/phpunit_bridge.html#usage :

Value | Recommended situation
-- | --
max[total]=0 | Recommended for actively maintained projects with robust/no dependencies
max[direct]=0 | Recommended for projects with dependencies that fail to keep up with new deprecations.
max[self]=0 | Recommended for libraries that use the deprecation system themselves and cannot afford to use one of the modes above.

It seems to me that, for us, `max[direct]=0` is not only a workaround here, but a good choice overall.

I vote to merge the PR this way.